override-locales/malte.json

```diff
@@ -0,0 +1,445 @@
+{
```

Comment:
Not sure whether we should include pl, ro and hr here as the maltesers do not use these languages. Included now for consistency.